PyTorch/vLLM_Tutorials/Deploying_vLLM/README.md

```diff
@@ -201,10 +205,31 @@ docker run -it --rm \
     -p 8000:8000 \
     -e MODEL=meta-llama/Llama-3.1-70B-Instruct \
     -e TENSOR_PARALLEL_SIZE=8 \
+    -e DTYPE=bfloat16 \
     -e MAX_MODEL_LEN=8192 \
     --name vllm-server \
     vllm-v0.7.2-gaudi-ub24:1.21.1-16
 ```
+2.2) Example for bringing up a vLLM server with a custom max model length, tensor parallel (TP) size and **DTYPE=fp8**.
```

Comment:
Add some info about how we first run calibration for fp8 on dataset included in docker. Also mention extra param for measurement directory provided as docker volume, and the behavior of calibration (skip) when measurement file is found.